### ISSUE TITLE
Algolia Search

### ISSUE THREAD

**Issue by bnomei** [k-next/plugins#58]
would love to have a port of
https://github.com/getkirby-plugins/algolia-plugin

**Comment by lukasbestle** [comment]
Yep, it's also not that much work to port over. We haven't discussed what we are going to do with our official plugins yet, but Algolia will be a pretty likely candidate for a v3 port.

**Comment by lukasbestle** [comment]
@bastianallgeier What is your opinion on the official plugins?

**Comment by bastianallgeier** [comment]
We will definitely port the Algolia plugin! We need it ourselves :)

**Comment by bastianallgeier** [comment]
I actually already started.

**Comment by bnomei** [comment]
@bastian any progress you can share? would love to do some testing.

**Comment by bnomei** [comment]
@bastianallgeier @lukasbestle any public progress?
